server/routerlicious/packages/services-client/src/storageUtils.ts

```diff
@@ -214,3 +216,42 @@ export function convertWholeFlatSummaryToSnapshotTreeAndBlobs(
 		sequenceNumber,
 	};
 }
+
+/**
+ * Converts existing IWholeSummaryTree to ISummaryTree
+ * @param wholeSummaryTree - wholeSummaryTree used on the payload for creating and uploading a document.
+ * @returns Summary tree to be used when creating a new document.
+ */
+export function convertWholeSummaryTreeToSummaryTree(
```

Comment:
As a general concept, a summary could have a handle type - a reference to a sub-tree in previously submitted summary.
This is not supported by this function, so it's not generic function (to convert any summary).
We probably want to name it appropriately such that someone accidently does not use it for that.
Or better - I'd create a new interface for that, that excludes IWholeSummaryTreeHandleEntry type.

Reply:
I will rename it.